apiary.apib

```diff
@@ -3487,10 +3487,11 @@ Get details for a single work order.
                         + type: `product` (string)
                         + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
                     + quantity: 3 (number)
-            + images (array)
+            + images (array) - Only included with request parameter `includes=images`.
                 + (object)
                     + id: `a856642a-a476-4b69-8e42-e7d1badb2ce9` (string)
```

Comment:
Q: should we nest this one deeper for variations of the same images e.g. thumbnails?

Reply:
You mean `id` as a key in a new object, having an array of multiple objects with separate variations? That is possible but we mentioned this during the TA and @stivni made the remark we can perfectly return multiple objects with the same `id` but a different `variation`.

Reply:
Let's postpone this because we don't know if and when we will add variations (and sample base64 images).